internal/command/action.go

```diff
@@ -22,18 +24,24 @@ type ActionConfig struct {
 	Ignores     []string
 	IgnoreAll   bool
 	Environment string
+	VersionProvider bool		//<---------------------------
```

Comment:
Stray trailing `//`?

Reply:
Yeah I guess the whole branch has a bunch of debug info that needs stripping. With that said, this is the reason to maybe add a debug flag and have some debug output?